openapi/auth-server.yaml

```diff
@@ -518,7 +518,6 @@ components:
       required:
         - type
         - actions
-        - identifier
```

Comment:
I think in the grant request we can still keep this as a required, no?

Token introspection, we will just have a unique case where the `identifier` is not required when calling this route